README.md

```diff
@@ -185,7 +185,11 @@ docker pull oneflowinc/oneflow:nightly-cuda11.7
     ```
     make -j$(nproc)
     ```
-
+    or run:
+    ```
+    ninja -j$(nproc)
```

Comment:
by default we don't recommend using ninja because make is widely available.

Reply:
> by default we don't recommend using ninja because make is widely available.

because this mlir-cuda is built by ninja not make ,the old document is outdated